examples/aml/main.go

```diff
@@ -28,7 +28,8 @@ func main() {
 
 	client = &yoti.Client{
 		SdkID: sdkID,
-		Key:   key}
+	}
+	client.LoadPEM(key)
```

Comment:
Would it be clearer to provide the key like this?
```go
key, err := yoti.LoadPEM(source)

client = &yoti.Client{
    SdkID: sdkID,
    Key:  key
}
```

Reply:
I think it would be better to do this and be explicit about what the function is doing. Although we might want to call it a new name actually if clients are passing `key` into the struct as before, but the type is different